extensions/git/package.json

```diff
@@ -2571,6 +2571,7 @@
           "group": "worktrees@1"
         },
         {
+          "when": "scmProviderContext == worktree",
           "command": "git.deleteWorktree",
```

Comment:
The label for the command should be `Delete Worktree` instead of `Delete Worktree...`. We use `...` in the command name as an indication that a picker will be shown which is not the case here.